src/app/components/Form/Form.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+type FormProps = {
+  nameLabel: string;
+  evaluationLabel: string;
+  onSubmit: (pupil: { name: string; evaluation: string }) => void;
+  submitted: boolean;
+};
+
+export default function Form({
+  nameLabel,
+  evaluationLabel,
+  onSubmit,
+}: FormProps): JSX.Element {
+  const [name, setName] = useState('');
+  const [evaluation, setEvaluation] = useState('');
+  const [nameSubmitted, setNameSubmitted] = useState(false);
+  const [evaluationSubmitted, setEvaluationSubmitted] = useState(false);
+  console.log(`nameSubmitted: ${nameSubmitted}`);
+  console.log(`evaluationSubmitted: ${evaluationSubmitted}`);
+
+  function handleSubmit(event: React.FormEvent<HTMLFormElement>) {
+    event.preventDefault();
+
+    if (name === '') {
+      setNameSubmitted(!nameSubmitted);
+      return;
+    } else if (evaluation === '') {
+      setEvaluationSubmitted(!evaluationSubmitted);
+      return;
```

Comment:
It's probably easier to have an `error` state (boolean) which is set to true if at least one field is not filled in. That way, you can replace your conditions for `nameSubmitted` and `evaluationSubmitted` in the returned JSX with a condition for the error state, respectively.